clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/ingestion/heartbeat/HeartbeatVersionedStats.java

```diff
@@ -23,4 +34,42 @@ public void recordLeaderLag(String storeName, int version, String region, long l
   public void recordFollowerLag(String storeName, int version, String region, long lag) {
     getStats(storeName, version).recordFollowerLag(region, lag);
   }
+
+  @Override
+  public synchronized void loadAllStats() {
+    Set<String> stores = getReportedStores();
+    metadataRepository.getAllStores().forEach(store -> {
```

Comment:
Yeah that's right, but that's ok.  We don't really want to start reporting until Helix assigns a state transition.  When the reporting thread runs it'll invoke the right methods for getting the stat onboarded, loadAllStats is just being proactive at start up time.  That's fine, but it's not desirable for this case.

Reply:
Hmm... but it looks like `loadAllStats` is ONLY called by the constructor, so shall we make it a no-op, rather than have any code at all in it? Or maybe I'm not understanding correctly what the goal of this override is 🤔

Reply:
That would probably be legit to do.  I'll do that.